xapian-applications/omega/templates/query

```diff
@@ -153,3 +153,4 @@ $map{$cgilist{B},<input type=hidden name=B value="$html{$_}">}
 </body>
 </html>
 $log{query.log}
+$log{search.log,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}$.$csv{$query,1}$.$csv{$hitlist{$id$if{$ne{$hit,$sub{$last,1}},$.}},1}$.$topdoc}
```

Comment:
No, that is correct. Sorry about the confusion. Not sure why I wrongly said that query IDs are not logged in `clicks.log`.

I think for addressing issue (b), `$cgi{QUERYID}` could be used here instead of calculating the hash a second time.